src/pages/PrescriptionPage.js

```diff
@@ -4,213 +4,189 @@
  * Sustainable Solutions (NZ) Ltd. 2019
  */
 
-import React, { useCallback } from 'react';
+import React from 'react';
+
 import PropTypes from 'prop-types';
-import { View } from 'react-native';
+
+import { View, Text } from 'react-native';
 import { connect } from 'react-redux';
 
-import { MODAL_KEYS } from '../utilities';
-import { useRecordListener } from '../hooks';
-import { getItemLayout, getPageDispatchers } from './dataTableUtilities';
-import { ROUTES } from '../navigation/constants';
-
-import { BottomConfirmModal, DataTablePageModal } from '../widgets/modals';
-import { PageButton, PageInfo, SearchBar, DataTablePageView } from '../widgets';
-import { DataTable, DataTableHeaderRow, DataTableRow } from '../widgets/DataTable';
-
-import { buttonStrings, modalStrings } from '../localization';
-import globalStyles from '../globalStyles';
-import { PageActions } from './dataTableUtilities/actions/index';
-
-export const Prescription = ({
-  dispatch,
-  data,
-  dataState,
-  pageObject,
-  sortBy,
-  isAscending,
-  modalKey,
-  hasSelection,
-  keyExtractor,
-  searchTerm,
-  modalValue,
-  columns,
-  getPageInfoColumns,
-  refreshData,
-  onSelectNewItem,
-  onEditComment,
-  onFilterData,
-  onDeleteItems,
-  onDeselectAll,
-  onCloseModal,
-  onCheck,
-  onUncheck,
-  onSortColumn,
-  onEditTotalQuantity,
-  onAddTransactionItem,
-}) => {
-  const { isFinalised, comment, prescriber } = pageObject;
-
-  // Listen for this invoice being finalised which will prune items and cause side effects
-  // outside of the reducer. Reconcile differences when triggered.
-  useRecordListener(refreshData, pageObject, 'Transaction');
-
-  const pageInfoColumns = useCallback(
-    getPageInfoColumns(pageObject, dispatch, ROUTES.PRESCRIPTION),
-    [comment, prescriber, isFinalised]
+import {
+  FavouriteStarIcon,
+  TableShortcut,
+  TableShortcuts,
+  Wizard,
+  SimpleTable,
+  PageButton,
+} from '../widgets';
+
+import { UIDatabase } from '../database';
+import { getColumns } from './dataTableUtilities';
+
+import { selectItem, selectPrescriber, switchTab } from '../reducers/PrescriptionReducer';
+
+/**
+ * File contains Four components for the PrescriptionPage. The container component Prescription and
+ * three "Tab" components PrescriberSelect/ItemSelect/Summary.
+ */
+
+const ALPHABET = [...'abcdefghijklmnopqrstuvwxyz'];
```

Comment:
perhaps move to a constants file?
We talked about localisation at one point and decided that the latin character set is fine, right? We'll have a look at localised versions later..

Reply:
honestly, I couldn't decide where to put the constants folder so I didn't make one. I've put it in `src/widgets/constants.js` - not sure 🤷‍♂ 

ff5c937e5c51cbd8bdf7ce2cbbaca2352663456c